### PR TITLE
Fully automate CI Framework with emcbot account

### LINKED ISSUE #1565: Update CI to use the EMC bot account
**Description**
Automated CI interactions should use the purpose-made [emcbot](https://github.com/emcbot) account instead of a personal account.

**Requirements**
All automated tag changes and results posting done by emcbot.

### LINKED ISSUE #1616: move the main CI driver to CRON to completely automate the process
As a develper/user we need the driver script to be CRON enabled so developers can issue a GitHub PR tag and remotely launch a CI test run without contacting any admin.

**Requirements**

On Orion and Hera:

The `devlop.sh` script needs to authenticate to GitHub CLI and we need a solution to authenticate with users who have this capability.

Ideally we need GitHub CLI installed on the system (currently found in admin's `~/bin/gh`)

**Acceptance Criteria (Definition of Done)**

An authorized user launches a CI test without an admin being contacted at all.

### PR DESCRIPTION
**Pull request by TerrenceMcGuinness-NOAA**
**Description**

This PR fixes logic in the main driver so it can function in cron along with the check and run scripts.
It also runs with the emcbot account so messaging to the user on the status to GitHub will come from emcbot

Fixes #1616 
Fixes #1565

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

- [x] Clone and Build tests with CI framework using labels only (no admin intervention)
- [x] Cycled test on Orion and Hera
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

**Review by WalterKolczynski-NOAA — APPROVED** [review]

File: ci/scripts/driver.sh [hunk 1]
```diff
@@ -109,8 +114,8 @@ for pr in ${pr_list}; do
     for yaml_config in "${HOMEgfs}/ci/cases/"*.yaml; do
       pslot=$(basename "${yaml_config}" .yaml) || true
       export pslot
-      sed -i "/^base:/a\  ACCOUNT: \${SLURM_ACCOUNT}" "${pr_dir}/global-workflow/parm/config/gfs/yaml/defaults.yaml"
-      sed -i "/^base:/a\  ACCOUNT: \${SLURM_ACCOUNT}" "${pr_dir}/global-workflow/parm/config/gefs/yaml/defaults.yaml"
+      HOMEgfs_ci="${HOMEgfs}"
+      export HOMEgfs_ci
```
Comment (aerorahul) [inline ci/scripts/driver.sh:118]: sorry, but remind me the purpose of this?

**Review by aerorahul — APPROVED** [review]
looks good to me.
just one question in the comments.

File: ci/scripts/driver.sh (hunk 1 above)
Comment (TerrenceMcGuinness-NOAA) [reply ci/scripts/driver.sh:118]: I need the full path to the yaml file to update the defaults to nems from the HOME ROOT of the repo where the CI code is .. in the driver script its HOMEgfs and I move it to the BASH env to HOMEgfs_ci and it gets resolved in the case yaml file [C96_atm3DVar.yaml](https://github.com/NOAA-EMC/global-workflow/blob/49983fab24c77470b82e7b5b8efc5cb1fb4e50f9/ci/cases/C96_atm3DVar.yaml)

Comment (TerrenceMcGuinness-NOAA) [reply ci/scripts/driver.sh:118]: recall we can not use HOMEgfs in the BASH because that will crash setup_expt.py thus the _ci tag at the end.

**Review by TerrenceMcGuinness-NOAA — COMMENTED** [review]
checking in comments

**Pull request merged**